mrjob/options.py

```diff
@@ -285,6 +282,74 @@ def add_hadoop_opts(opt_group):
             help='Deprecated alias for --hadoop-tmp-dir'),
     ]
 
+# TODO - mtai @ davidmarin - pulled common dataproc/emr opt here
+def add_dataproc_emr_opts(opt_group):
+    return [
+        opt_group.add_option(
+            '--bootstrap', dest='bootstrap', action='append',
+            help=('A shell command to set up libraries etc. before any steps'
+                  ' (e.g. "sudo apt-get -qy install python3"). You may'
+                  ' interpolate files available via URL or locally with Hadoop'
+                  ' Distributed Cache syntax ("sudo dpkg -i foo.deb#")')),
+
+        opt_group.add_option(
+            '--bootstrap-python', dest='bootstrap_python',
+            action='store_true', default=None,
+            help=('Attempt to install a compatible version of Python'
+                  ' at boostrap time. Currently this only does anything'
+                  ' for Python 3, for which it is enabled by default.')),
+    ]
+
+def add_dataproc_opts(opt_group):
+    """Options for ``dataproc`` runner"""
+    return [
+        opt_group.add_option(
+            '--gcp-project', dest='gcp_project', default=None,
+            help=('Project to run Dataproc jobs in.')),
+
+        opt_group.add_option(
+            '--gcp-region', dest='gcp_region', default='global',
+            help=('Region to run Dataproc jobs in. Default is global')),
+
+        opt_group.add_option(
+            '--gcp-zone', dest='gcp_zone', default=None,
+            help=('Zone to run Dataproc jobs in.  ')),
+
+        # TODO - mtai @ davidmarin - collapse cluster_id
+        opt_group.add_option(
+            '--gcp-cluster', dest='gcp_cluster',
+            help=('Dataproc cluster name - REQUIRED')),
+
+        # instance types
+        # TODO - mtai @ davidmarin - collapse with ec2_instance_type
```

Comment:
Great, glad you're already thinking about this.

I think the Dataproc runner should just use general names for things (e.g. `num_instances`, `image_version`) from the get-go, and we should move the EMR runner over to these names.